src/shared/components/feature-summary-strip/VariantSummaryStrip.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+
+import useResizeObserver from 'src/shared/hooks/useResizeObserver';
+
+import VariantConsequence from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-consequence/VariantConsequence';
+import VariantAllelesSequences from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-alleles-sequences/VariantAllelesSequences';
+import VariantLocation from 'src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-location/VariantLocation';
+
+import type { FocusVariant } from 'src/shared/types/focus-object/focusObjectTypes';
+
+import styles from './FeatureSummaryStrip.scss';
+
+const MEDIUM_WIDTH = 720;
+
+enum Display {
+  FULL = 'full',
+  MINIMAL = 'minimal'
+}
+
+type Props = {
+  variant: FocusVariant;
+  isGhosted?: boolean;
+};
+
+type WidthAwareProps = Props & {
+  display: Display;
+};
+
+const VariantSummaryWrapper = (props: Props) => {
```

Comment:
It shouldn't be necessary to extend the focus variant with all the display-related stuff that we need to show; let's only use the focus variant to keep the minimal information about which variant is in focus. Use the same query as the `TrackPanelVariant` component is using to get the necessary variant data.

Reply:
✅